public/app/features/alerting/unified/utils/time.ts

```diff
@@ -99,3 +99,7 @@ export function parsePrometheusDuration(duration: string): number {
 
   return totalDuration;
 }
+
+export const isNullDate = (date: string) => {
```

Comment:
I guess we know that null dates will come with this value?

Reply:
Yes, I verified that in some cases we get those kind of dates (like for paused alert rules) and trying to format them would display something completely wrong